src/components/TruckSizeSelect/TruckSizeSelect.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import { propTypes } from 'react-bootstrap/esm/Image';
+import Inputs from "../../models/Inputs";
+import "./TruckSizeSelect.css"
+
+const sizes: [Inputs["truckSize"], Inputs["truckSize"], Inputs["truckSize"]] = ['small', 'medium', 'large']
+
+interface Props {
+    onUpdate: (truckSize: Inputs["truckSize"]) => void;
+}
+
+const TruckSizeSelect = ({ onUpdate }: Props) => {
+    const [selectedSize, setSelectedSize] = useState<Inputs["truckSize"]>('small');

```

Comment:
I think we should not have a local state here, but fetch the state all the way from the root so that we have only one source of truth (so in that case you would push down `Inputs` as another prop all the way down here). What do you think.

Reply:
Yes, that is a better idea ^^